src/app/payment/paymentMethod/MasterpassPaymentMethod.tsx

```diff
@@ -20,17 +21,19 @@ const MasterpassPaymentMethod: FunctionComponent<MasterpassPaymentMethodProps &
         },
     }), [initializePayment]);
 
+    const userLocale = getUserLocale();
```

Comment:
Yes as, @TomA-R said, can you follow the advice I gave to @BC-AxelMalagon in https://github.com/bigcommerce/checkout-sdk-js/pull/1168#discussion_r661892982. For Checkout JS, what you need to do here is to use to simply call `language.getLocale()`. `language` instance is already in the scope of the function so you can just use it. I believe you still need to transform the value somehow in the format Masterpass expects (i.e.: `_` instead of `-`)